pmlc/conversion/gpu_to_spirv/gpu_to_spirv.cc

```diff
@@ -31,6 +31,22 @@ struct StdxSubgroupBroadcastOpConversion final
     : public SPIRVOpLowering<stdx::SubgroupBroadcastOp> {
   using SPIRVOpLowering<stdx::SubgroupBroadcastOp>::SPIRVOpLowering;
 
+  LogicalResult
```

Comment:
Maybe template on the op to generate to reduce code duplication

Reply:
Done.